exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return numbers[0]; // complete this statement
```

Comment:
Not quite right. Please use the `arr` parameter provide in the function `first`.